services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubRepository.java

```diff
@@ -1,7 +1,12 @@
+/**
```

Comment:
Ok. I added these copyright notices on those files based on https://github.com/redhat-kontinuity/catapult/blob/master/services/github-service-impl/src/main/java/org/kontinuity/catapult/service/github/GithubResource.java and because that's what we've been doing on JBoss Tools, but I'm fine if they are not needed.